### ISSUE TITLE
I got nan losses on results

### ISSUE THREAD

**Issue by LIUTAOGE** [microsoft/scene_graph_benchmark#95]
INFO:maskrcnn_benchmark.trainer:eta: 10:55:48  iter: 5100  loss: nan (nan)  loss_obj_classifier: 0.0000 (0.0000)  loss_pred_classifier: nan (nan)  time: 1.1246 (1.1275)  data: 0.0462 (0.0644)  lr: 0.000467  max mem: 9431

SOLVER:
  BASE_LR: 0.001
  WEIGHT_DECAY: 0.0001
  MAX_ITER: 40000
  STEPS: (50000,)
  IMS_PER_BATCH: 16
  CHECKPOINT_PERIOD: 10000

**Comment by adnenabdessaied** [comment]
You might want to adjust the learning rate according to the batch size you are using following https://github.com/facebookresearch/Detectron/blob/main/configs/getting_started/tutorial_1gpu_e2e_faster_rcnn_R-50-FPN.yaml#L14

**Comment by 184446223** [comment]
> 您可能想根据您使用的批量大小调整学习率，如下https://github.com/facebookresearch/Detectron/blob/main/configs/getting_started/tutorial_1gpu_e2e_faster_rcnn_R-50-FPN.yaml#L14

Hello, can you share the pretrain model? The original link has expired.
